Gemfile

```diff
@@ -10,4 +10,6 @@ end
 
 group :development do
   gem 'github_changelog_generator', '>= 1.16.4'
+  # provides the rubocop config and an awesome circular dependency to boot!
```

Comment:
:raised_eyebrow: Wat?  CI seems happy with this…

Reply:
`voxpupuli-test` has a dep on `facterdb`.  I'm not sure if this would work at all if the Gemfile was in a released version of `facterdb`.  We all use constraint resolvers every day but does anyone actually understand them? :)